darts/tests/utils/test_formatting.py

```diff
@@ -0,0 +1,117 @@
+from darts.utils._formatting import (
+    format_bytes,
+    format_dict,
+    format_list,
+    make_collapsible_section,
+    make_paragraph,
+    truncate_key,
+)
+
+
+class TestFormatting:
+    def test_format_bytes(self):
+        """Test format_bytes with different units."""
+        assert format_bytes(500) == "500.00 B"
+        assert format_bytes(1024) == "1.00 KB"
+        assert format_bytes(1024 * 1024) == "1.00 MB"
+        assert format_bytes(1024 * 1024 * 1024) == "1.00 GB"
+
+    def test_format_bytes_precision(self):
+        """Test format_bytes with custom precision."""
+        assert format_bytes(1536, precision=1) == "1.5 KB"
+
+    def test_truncate_key(self):
+        """Test truncate_key function."""
+        assert truncate_key("hello") == "hello"
+        assert truncate_key("a" * 20, max_len=20) == "a" * 20
+        assert truncate_key("a" * 21, max_len=20) == "a" * 17 + "..."
+        assert len(truncate_key("a" * 30, max_len=20)) == 20
+
+    def test_format_dict_empty(self):
+        """Test format_dict with empty dict."""
+        assert format_dict({}) == "    <empty>"
+        assert format_dict({}, render_html=True) == "&lt;empty&gt;"
+
+    def test_format_dict_items(self):
+        """Test format_dict with items."""
+        d = {"key1": "value1", "key2": "value2"}
+        result = format_dict(d)
+        assert "    key1" in result
+        assert "value1" in result
+        assert "\n" in result
+
+    def test_format_dict_truncation(self):
```

Comment:
same comment here and in other tests below (you can also pick lower "max" display values to check against shorter expected strings